include/test/CDataFrameAnalysisSpecificationFactory.h

```diff
@@ -117,6 +118,7 @@ class TEST_EXPORT CDataFrameAnalysisSpecificationFactory {
     TRestoreSearcherSupplier* m_RestoreSearcherSupplier = nullptr;
     // Classification
     std::size_t m_NumberClasses = 2;
+    std::size_t m_NumberTopClasses = 1;
```

Comment:
I will change it, but I am defaulting to `1` as that is how it was before my change.

Reply:
Thanks! It does feel like it probably ought to match the default behaviour (although this may have some knock on effects). If you are keeping as is feel free to leave as is. This needn't be part of this PR in that case.